src/components/log/list/LogList.tsx

```diff
@@ -58,6 +60,13 @@ function LogListBare({ data, ...displayOpts }: LogListProps) {
               >
                 {message}
               </Button>
+              <Button
```

Comment:
Good idea allowing to show all instead of paging through. But need to polish the implementation a bit 
<img width="342" height="107" alt="image" src="https://github.com/user-attachments/assets/951cd61d-58b4-4e33-814b-00d10522e608" />